content/Watsonx/WatsonxAI/100.md

```diff
@@ -15,6 +15,89 @@ toc: true
 
 ## FAQs
 
+## How do I access a watsonx.ai instance?
+
+### How to acquire a watsonx.ai instance from Techzone (TZ)
+
+
+To gain access to a watsonx.ai instance from TZ, navigate to this link [here](https://ibm.biz/WATSONX-AI-TZ). Once there:
+
+1. Select **Reserve now**
+     ![](images/100/reserve-now-btn.png)
+
+2. Change the Name of the reservation to *your name* watsonx.ai L3 SaaS (1).
+3. Select **Practice/Self-Education** (2).
+4. For the Purpose description, enter `Completing L3 Lab`(3).
+5. Select your preferred Geography – here `us-south` is chosen (4).
+
+      ![](images/100/reservation-type.png)
+
+Continue to the bottom part of the reservation page:
+
+6. Specify the end date and time (1).
+7. Select Dedicated (should be the default) service (2).
+8. Select No (should be the default) for installing Db2 (3).
+9. On the right-hand side at the bottom, accept the Terms and Conditions, then click **Submit** (4).
+
+      ![](images/100/submit-reservation.png) 
+
+You will get an email sent to your IBMid email address letting you know that your reservation is being provisioned (this should happen within 10-15 minutes). 
+
+Once the watsonx.ai provisioning is completed, you will receive a second email telling you that it is ready for use. 
+
+This email looks like the example below. You will need to click on the **HERE** link (as highlighted below) to go and accept the invitation.
+
+   ![](images/100/ready-email.png)
+
+Clicking on **HERE** brings you to a login page where you need to provide your IBMid (and may have to log in with your credentials if you have not already done so). 
+
+10. Then you will see the Action required item. Click on the item.
+11. The following page opens up. Click **Join now**.
+
+      ![](images/100/join-now.png)
+
+12. You will now be sent back to the IBM Cloud login page. Use your IBMid to log in. You are asked to join the TechZone account. Accept the Terms and Conditions, then click **Join Account**.
+
+13. When you log in, go to the menu bar on the top right and select the `itz-watsonx` account.
+
+      ![](images/100/itz-watsonx.png)
+
+14. In the search field, enter `watsonx`. The search will return a list similar to the following. Click `watsonx`.
+
+      ![](images/100/search-watsonx.png)
+
+15. You will now see the AI and data platform page. Click `Get started` from the watsonx.ai tile.
```

Comment:
For some reason, the button says 'Launch' for me.

Reply:
This button may instead say **Launch** if you have provisioned a watsonx instance.